lib/child-writer/index.js

```diff
@@ -126,8 +126,22 @@ exports.write = (image, drive, options) => {
    * }));
    */
   const bridgeRobotMessage = (message) => {
+    const ROBOT_COMMAND_LOG = 'log';
+
     try {
-      const parsedMessage = robot.parseMessage(message);
+      const parsedMessage = _.attempt(() => {
```

Comment:
Since this is wrapped in an `_.attempt()`, I think this can go outside of the try-catch-block